datasource/api/src/main/java/org/opennms/oce/datasource/api/Alarm.java

```diff
@@ -46,4 +48,5 @@
 
     String getDescription();
 
+    Set<Alarm> getRelatedAlarms();
```

Comment:
In this project, we use separate entities for alarms and situations (i.e. incidents). The OpennmsDatasource distinguishes the two and add situations to a separate stream: https://github.com/OpenNMS/oce/blob/master/datasource/opennms/src/main/java/org/opennms/oce/datasource/opennms/OpennmsDatasource.java#L240

Can we avoid adding the notion of related alarms to the Alarm interface and use the Incident interface instead?